_posts/2016-8-8-this-week-in-rust-docs-16.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: This Week in Rust Docs 16
+date: 2016-08-08
+---
+
+Hello and welcome to *This Week in Rust Docs*!
+
+*This Week in Rust Docs* is openly developed [on GitHub](https://github.com/GuillaumeGomez/this-week-in-rust-docs).
+If you find any errors in this week's issue, [please submit a PR](https://github.com/GuillaumeGomez/this-week-in-rust-docs/pulls).
+
+And of course, don't forget to look at the docs:
+
+* [Stable](https://doc.rust-lang.org/)
+* [Beta](http://doc.rust-lang.org/beta/)
+* [Nightly](http://doc.rust-lang.org/nightly/)
+
+This week's edition was edited by: [GuillaumeGomez](https://github.com/GuillaumeGomez).
+
+# Latest news
+
+The [doc team RFC](https://github.com/rust-lang/rfcs/pull/1683#issuecomment-237384575) has entered its final comment period!
+
+A new RFC has been opened: [Add API documentation front page styleguide](https://github.com/rust-lang/rfcs/pull/1687).
+
+A topic to propose crates for the Rust Doc Days has been created [here](https://users.rust-lang.org/t/call-for-proposals-for-next-rust-doc-days-crates/6685). Please take a look!
+
+# Current opened issues
+
+For now, here are the three big issues for Rust documentation:
+
+* [Error code list which need to be updated to new format](https://github.com/rust-lang/rust/issues/35233)
+* [The Standard Library Documentation Checklist](https://github.com/rust-lang/rust/issues/29329)
+* [Add error explanations for all error codes](https://github.com/rust-lang/rust/issues/32777)
+
+They all need help to move forward so any contribution is very welcome!
+
+There are currently around 50 other documentation issues opened. Look for [A-docs](https://github.com/rust-lang/rust/issues?q=is%3Aopen+is%3Aissue+label%3AA-docs) tagged issues on github!
+
+# Call for participation
+
+There's now a call for participation to [display all methods of a type, even those from implicit traits in rustdoc](https://github.com/rust-lang/rust/issues/33772). This is a great way to help users find everything that a type can do. Any help on it would be very appreciated!
```

Comment:
Maybe link to <http://www.jonathanturner.org/2016/08/helping-out-with-rust-errors.html> as well?